test/escrow_spec.js

```diff
@@ -328,4 +327,84 @@ contract("Escrow", function() {
     assert.equal(balanceBeforeCreation, balanceAfterCancelation, "Invalid seller balance");
 
   });
+
+  describe("Rating a released Transaction", async() => {
+    let receipt, created, escrowId;
+
+    beforeEach(async() => {
+      const isPaused = await Escrow.methods.paused().call();
+      if (isPaused) {
+        receipt = await Escrow.methods.unpause().send({from: accounts[0]});
+      }
+      receipt = await Escrow.methods.create(accounts[1], expirationTime).send({from: accounts[0], value});
+      created = receipt.events.Created;
+      escrowId = created.returnValues.escrowId;
+      await Escrow.methods.release(escrowId).send({from: accounts[0]});
+    })
+
+    it("should not allow a score last less than 1", async() => {
+      try {
+        receipt = await Escrow.methods.rate_transaction(escrowId, 0).send({from: accounts[1]});
```

Comment:
Do you really need to override the one above? You don't really reuse the receipts so you could keep them local.